libraries/Microsoft.Bot.Builder.Core/BotContext.cs

```diff
@@ -10,15 +10,15 @@
 
 namespace Microsoft.Bot.Builder
 {
-    public class BotContext : FlexObject, IBotContext
+    public class BotContext : IBotContext
```

Comment:
>BotContext [](start = 17, length = 10)

I renamed this TurnContext in my implementation as I thought we had decided that better represents that it live for the life of a turn. I'm not opposed to renaming it back.

Reply:
I changed my name back to `BotContext` as it doesn't seem like anyone likes `TurnContext`.